hmda/src/main/scala/hmda/persistence/submission/HmdaValidationError.scala

```diff
@@ -576,7 +576,7 @@ object HmdaValidationError
       }
 
     for {
-      macroEdits <- macroValidation(larSource)
+      macroEdits <- macroValidation2018(larSource)
```

Comment:
Rather than always using `macroValidation2018`, we should extract the submission year from `submissionId` and use the appropriate years' macrovalidation.